x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph_investigation/graph_investigation.stories.test.tsx

```diff
@@ -51,6 +77,31 @@ describe('GraphInvestigation Component', () => {
     expect(getAllByText('~ an hour ago')).toHaveLength(2);
   });
 
+  it('shows error on bad kql syntax', async () => {
+    const mockDangerToast = action('notifications:addError');
+    const { getByTestId } = renderStory();
+
+    // Act
+    const queryInput = getByTestId('queryInput');
+    await userEvent.type(queryInput, '< > sdg $@#T');
+    const querySubmitBtn = getByTestId('querySubmitButton');
+    querySubmitBtn.click();
+
+    // Assert
+    expect(mockDangerToast).toHaveBeenCalledTimes(1);
+  });
+
+  it('calls refresh on submit button click', () => {
+    const mockRefresh = action('refresh');
```

Comment:
For better maintainability, we should define a central place where we store Storybook actions in a constant, otherwise, as it grows it can become tricky to understand where actions such as `refresh` and `notifications:addError` are coming from